docs/content/how-to/disaster-recovery/scenarios/same-cluster-restore.md

```
@@ -0,0 +1,186 @@
+| ---------- | --------------------- | --------- |
+| **AWS** | Fix OIDC Identity Provider | Run `hypershift fix dr-oidc-iam`. See [AWS Platform Guide](../platform-guides/aws.md#fixing-oidc-after-restore). |
+| **Azure** | No additional steps | — |
+| **Agent / Bare Metal** | Verify InfraEnv and agents | Ensure agents can reach the InfraEnv Discovery ISO. See [Agent Platform Guide](../platform-guides/agent.md). |
+| **KubeVirt** | Wait for VM recreation | VMs are automatically recreated. See [KubeVirt Platform Guide](../platform-guides/kubevirt.md). |
+| **OpenStack** | No additional steps | — |
+
+## Using the HyperShift CLI
+
+Instead of manually creating backup and restore manifests, you can use the HyperShift CLI:
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)